var/spack/repos/builtin/packages/nextflow/package.py

```diff
@@ -7,23 +7,27 @@
 
 
 class Nextflow(Package):
-    """Data-driven computational pipelines"""
+    """Data-driven computational pipelines."""
 
     homepage = "https://www.nextflow.io"
-    url = "https://github.com/nextflow-io/nextflow/releases/download/v0.24.1/nextflow"
+    url      = "https://github.com/nextflow-io/nextflow/releases/download/v21.04.3/nextflow-21.04.3-all"
 
-    version('20.10.0', sha256='54f76c83cbabe8ec68d6a878dcf921e647284499f4ae917356e594d873cb78dd', expand=False)
-    version('20.07.1', sha256='de4db5747a801af645d9b021c7b36f4a25c3ce1a8fda7705a5f37e8f9357443a', expand=False)
-    version('0.25.6', sha256='9498806596c96ba87396194fa6f1d7d1cdb739990f83e7e89d1d055366c5a943', expand=False)
-    version('0.24.1', sha256='0bfde5335b385e3cff99bf4aab619e583de5dc0849767240f675037a2e7c1d83', expand=False)
-    version('0.23.3', sha256='ffe1c314962ff97ebf47b0567883e152522acfbf6fd5800200b1a7a0ca2896d2', expand=False)
-    version('0.21.0', sha256='076089079479da0d91fe1ad7aad06816164ecbcf17f73c55e795b1db8462b28d', expand=False)
-    version('0.20.1', sha256='02635f3371f76a10e12f7366508c90bacf532ab7c23ae03c895317a150a39bd4', expand=False)
-    version('0.17.3', sha256='05563ee1474fbef22f65fa3080792dcb08d218dd1b1561c517ebff4346559dbe', expand=False)
+    maintainers = ['dialvarezs']
+
+    version('21.04.3', sha256='6eed23699055af60a85dd017b0351b7151982c6440b1ec5c273c1d4d07567572', expand=False)
+    version('20.10.0', sha256='ef6e229307f04a6abd27f06f7e999dc77e064ab94d0202243801f8b69bb3641c', expand=False)
```

Comment:
Why did this (and all others below) checksum change?

Reply:
I guess it's because of using "complete" binaries?

Reply:
Yes, it's because of that. I think is a better idea to use the full binaries because if not, basically every user using this as a module would have to download and "build" the full binary anyway

Reply:
What do you suggest?

Reply:
Unfortunaly I don't know much about this package, but maybe the previous contributors @antonl @baberlevi @peetsv @hartzell could comment on the choice to use "complete" vs "default" binaries?

Reply:
Maybe @pditommaso can shed some light here? Is there any "inconvenience" on using the "all" package?

Reply:
The main difference betweeen `nextflow` and `nextflow-all` is that the first download all required deps on the first launch, while the second is self-contained. However the latter does not allow the use of some external plugins. Therefore my suggesting would be to use `nextflow` executable.